manuscript.qmd

```diff
@@ -806,8 +806,8 @@ host genomes [@Qin2022], transcriptomes, or proteomes [@nyholm2020; @moreno-indi
 containers support complex combinations omics data sets [@huber2015; @ramos2017; @amezquita2020].
 Such integrative approaches in microbiome
 research can be broadly categorized into three groups: Machine Learning (ML)
-predictive models [@mallick2024], association-based models, and
-latent variable models [@Jeganathan2021; @moiz2025;
+predictive modelling [@mallick2024], association-based modelling, and
+latent variable modelling.  [@Jeganathan2021; @Sherwani2025;
```

Comment:
Repetitive. Fix in new PR:

Machine Learning (ML) predictive  [@mallick2024], association-based, and
latent variable modelling.